ash/src/vk/prelude.rs

```diff
@@ -0,0 +1,254 @@
+use crate::vk::definitions::BaseOutStructure;
```

Comment:
I am not super liking the name `prelude.rs` here. But I also have a hard time coming up with something better. Maybe the rest should move into `vk.rs`.